.cypress/integration/trace_analytics_test/trace_analytics_services.spec.js

```diff
@@ -201,7 +201,8 @@ describe('Testing Service map', () => {
     cy.get('[data-text="Errors"]').click();
     cy.contains('60%');
     cy.get('[data-text="Duration"]').click();
-    cy.contains('100');
```

Comment:
Is it okay to not check for this Duration value anymore?

Reply:
The duration value was just reading the 100 off the sidebar, cy.contains('Average duration (ms)'); is a more accurate check if switching the metrics worked. This is just to cover the rendering to replace the wait functionality though as this test is for the node details checked on line 210

cy.get('.euiText.euiText--small').contains('Average duration: 216.43ms').should('exist');

Reply:
Thanks, sounds good to me!